chain/starknet/src/codec.rs

```diff
@@ -0,0 +1,32 @@
+#[allow(clippy::all)]
```

Comment:
Do we really need this?

Reply:
Oops must have been mixed into this branch when working on #4169.

Reply:
Removed.